UefiPayloadPkg/SPI/SPI.inf

```diff
@@ -0,0 +1,75 @@
+#/** @file
+#
+#  Component description file for SMMSTORE module
+#
+#  Copyright (c) 2020, 9elements Agency GmbH<BR>
+#
+#  SPDX-License-Identifier: BSD-2-Clause-Patent
+#
+#**/
+[Defines]
+  INF_VERSION                    = 0x00010005
+  BASE_NAME                      = SPI
+  FILE_GUID                      = c2c2e656-ee66-41e0-bee3-29c6f16f49c0
+  MODULE_TYPE                    = DXE_RUNTIME_DRIVER
+  VERSION_STRING                 = 1.0
+  ENTRY_POINT                    = SPIInitialize
+
+[Sources]
+  SPI.h
+  # SPI.c
```

Comment:
Not compiling it?

Reply:
SPI.h has no function definitions anymore.